src/frontend/src/main/SearchField/index.jsx

```diff
@@ -101,7 +101,16 @@ export default function SearchField({ onLoading, onTicketsData, setRequestBody }
   }
 
   function transformData(item) {
-    return { value: item.cityUkr, label: `${item.cityUkr}, ${item.country}` };
+    if (item.siteLanguage === 'ua' && item.cityEng !== null) {
+      return ({ value: item.cityUa, label: `${item.cityUa} (${item.cityEng}), ${item.country}` });
+    }
+    if (item.siteLanguage === 'ua' && item.cityEng === null) {
+      return ({ value: item.cityUa, label: `${item.cityUa}, ${item.country}` });
+    }
+    if (item.siteLanguage === 'eng' && item.cityUa === null) {
+      return ({ value: item.cityEng, label: `${item.cityEng}, ${item.country}` });
+    }
+    return ({ value: item.cityEng, label: `${item.cityEng} (${item.cityUa}), ${item.country}` });
```

Comment:
please, try to refactor this code to switch case like here - https://stackoverflow.com/questions/5464362/javascript-using-a-condition-in-switch-case

Reply:
or try to break into method like this

`if (item.siteLanguage === 'ua') {
    if (item.cityEng !== null) {
      return ({ value: item.cityUa, label: `${item.cityUa} (${item.cityEng}), ${item.country}`
      });
      ({ value: item.cityUa, label:`${item.cityUa}, ${item.country}`});
    }
  }
  if (item.siteLanguage === 'eng') {
    if (item.cityEng !== null) {
      return ({ value: item.cityEng, label:`${item.cityEng}, ${item.country}`});
    }
    ({ value: item.cityEng, label:`${item.cityEng} (${item.cityUa}), ${item.country}` })
  }`


but probably switch case would be better for a later adding new lang. as i18